docs/basics/final-words.md

```diff
@@ -1,32 +1,35 @@
 # Final words
 
-Congratulations, you made it this far!
+Congratulations, you completed the Earthly tutorial! 
 
-To learn more about Earthly, take a look at the [examples directory on GitHub](https://github.com/earthly/earthly/tree/main/examples), where you will find the complete code used in this guide:
+Learning Earthly does not stop here. Discover more of what Earthly can do by exploring the documentation.
 
+**Recommended reading:**
```

Comment:
Nit: might be better to have these as a subtitles?

```suggestion
##### Recommended reading:
```

Reply:
That makes sense will make the change.